### PR TITLE
Feature\kids-463: Family Goal Creation Start screen

### PR DESCRIPTION
**Pull request by pavelpavelko**
## Description
- Start page of the Family Goal Creation is implemented

### CODE REVIEW

File: lib/features/children/family_goal/widgets/family_goal_creation_stepper.dart [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+class FamilyGoalCreationStepper extends StatelessWidget {
+  const FamilyGoalCreationStepper({
+    required this.currentStep,
+    super.key,
+  });
+
+  final FamilyGoalCreationSteps currentStep;
+
+  // static const String flagSwallowtail = '\uf74c';
```
Comment (Daniela510) [inline lib/features/children/family_goal/widgets/family_goal_creation_stepper.dart:13]: Delete, we will fix font awesome pro icons in another way.

File: lib/features/children/family_goal/widgets/family_goal_creation_stepper.dart [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+                    //   flagSwallowtail,
+                    //   style: TextStyle(
+                    //     fontFamily: 'FontAwesome',
+                    //     color: currentStep.index >=
+                    //             FamilyGoalCreationSteps.confirmation.index
+                    //         ? AppTheme.givtLightGreen
+                    //         : AppTheme.familyGoalStepperGray,
+                    //     fontSize: 20,
+                    //   ),
+                    // ),
```
Comment (Daniela510) [inline lib/features/children/family_goal/widgets/family_goal_creation_stepper.dart:85]: Also not necessary so let's delete

File: lib/features/children/family_goal/widgets/family_goal_creation_stepper.dart [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+              ],
+            ),
+          ),
+          const SizedBox(height: 5),
+          Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              Text(
+                //TODO: POEditor
+                '1. Cause',
```
Comment (Daniela510) [inline lib/features/children/family_goal/widgets/family_goal_creation_stepper.dart:97]: Should we do POEditor before a merge, or deal with this later?

**Review by Daniela510 — APPROVED** [review]

File: lib/features/children/family_goal/pages/create_family_goal_page.dart [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        centerTitle: true,
+        title: Text(
+          //TODO: POEditor
+          'Create a Family Goal',
+          style: Theme.of(context).textTheme.titleLarge?.copyWith(
+                fontFamily: 'Mulish',
```
Comment (Daniela510) [inline lib/features/children/family_goal/pages/create_family_goal_page.dart:21]: Also we do not have the font 'Mulish' in this repo so it just defaults to basic sans serif (ugly). Either upload the mulish family ttfs manually to the fonts folder and pubspec or add it through google fonts

File: lib/features/children/family_goal/widgets/family_goal_creation_stepper.dart (hunk 3 above)
Comment (pavelpavelko) [reply lib/features/children/family_goal/widgets/family_goal_creation_stepper.dart:97]: I'm going to apply POEditor later on when the entire feature will be finished

**Comment by Daniela510** [comment]
Awesome!

**Pull request merged**